app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
Apply formatter, is it done in [example](https://mateacademy.slack.com/archives/C05HCQB4HQA/p1691763466046359)

Reply:
You can use black for doing this easier and faster